src/filters/dates/partialDates.test.ts

```diff
@@ -0,0 +1,143 @@
+import { DateTime } from 'luxon';
+import { TestCaseSchema } from '../../serina.schema';
+import PartialDates from './partialDates';
+
+const currentYear = DateTime.utc().year;
+
+const mockDates = (day, month, year) => DateTime.utc()
+    .set({ day, month, year })
+    .endOf('day')
+    .toJSDate();
+
+describe('Dates', () => {
```

Comment:
Don't think we need to have the outer Describe if you are going to create a new test file?

Reply:
Yes, forgot to remove whilst refactoring.

Reply:
So can we get rid of it?